docs/translation-guidelines.md

```diff
@@ -0,0 +1,43 @@
+
+
+## The "Cuttle feel"
+Some of the terminology refers to the seas and the ocean's depth, but it's not meant to be applied to every term in the game.  Most words used are just standard words used in card games in general. Each language should use words and expressions that feel right to native speakers and not use literal trnaslations with the only goal of using the same word/expression as in the English version.  
+
+
+## Definition of phisical entities
+
+- **Point card** :  Any card with a numeric value (1 to ten).
+- **Royal** :  Any card with a figure.  King, Queen or Jack.
+- **Deck** :  The pile of cards that have not been picked or distributed.
+- **Scrap pile** :  The pile of discarded cards.
+- **Hand** :  The cards a player hold in their hands.
+- **Field** :  The area where a player lays their 'active' card.
+- **Table** :  The whole area where cards are played containing the two fields.
+
+
+## Definition of non phisical entities
+
+- **Scuttle** :  When a player targets and scraps an opponent's point card.  Name and verb.
+- **One-off** :  The special capacity of a point card.  One-off refers to the idea of it being a one time action.  Royals don't have one-offs. Their capacity don't have a specific name.
+- **To target** :  To choose an opponent's card that will undergo an attack.
+- **To discard** :  To take a card, usually from your hand, and put it in the scrap pile.
+- **To pick** :  To take a card from the top of the deck.
+- **Stalemate** :  The game has reached a dead end, it then ends in a tie.
+- **Season** :  A period of time during wich the stats are compiled.
+- **Casual game** :  Standard game that has no effect on statistics.
+- **Ranked game** :  Competitive game from wich the outcome is compiled in the statistics of the current seaon.
+
+
+## Translation guide
+
+- **Scuttle** :  This word is meant to stay the same in every language.  It is used as a noun and a verb in English, but these two uses are not an obligation.  For languages with grammatical genders, the translator may use the one that 'sounds better' in their language.
+- **Scrap pile** :  This expression is not meant to transmit the idea of "a pile of junk or old metalic wastes".  So it should be translated to a word that refers to something where you discard - **things that you no longer need.  Many languages have a word for the pile of discarded cards in card games, that word is perfect.
+- **To srap a card** :  To get rid of a card by putting it in the scrap pile.  
+- **Deck, hand, discard, pick, royal,...** :  Most language already have a specific terminology for card games.  These words should be used.
+- **One-off** :  This refers to the idea of an action taking place once with no persistent effect.  If the target language has such a word/expression, it may be used. 
+  A basic literal translation (with Google translate) might sound weird.  In that case, it can be translated to something like "Capacity, Effect, tactical, super power, ...". 
```

Comment:
```suggestion
  A basic literal translation (with Google translate) of one-off might sound weird.  In that case, it can be translated to something like "Ability, Effect, super power, ...". 
```

Reply:
I'd like to hear your thoughts about "capacity" and "tactical".  I now realize how they don't have the same meaning in English and French.  For translating one-off, I used "tactical capacity".   That translates to "strategic ability".

Reply:
In english, strategic ability is reasonable (though special ability is probably closer to the feel), but 'tactical capacity' feels incorrect (not to say that's wrong for French). In English, 'capacity' is more of a generally aptitude or affinity for something rather than a specific skill. Like you can have a capacity for kindness or for some general thing but we wouldn't generally use 'capactiy' to describe a specific skill or ability